.prettierignore

```diff
@@ -0,0 +1 @@
+*.html
```

Comment:
I dont think we should be ignoring html files by prettier

Reply:
It's a tad illegible to me and makes coding so much slower. Have removed it as suggested but wont be using it in my editor and don't see why changing one's way of using git, html formatting should be enforced. 
![Screenshot 2020-03-15 at 13 51 29](https://user-images.githubusercontent.com/13540819/76702791-981ecd00-66c4-11ea-8366-7f07343c050b.png)
![Screenshot 2020-03-15 at 13 51 16](https://user-images.githubusercontent.com/13540819/76702800-a40a8f00-66c4-11ea-9d4b-4b4c69d92581.png)

Reply:
the good thing with prettier is it will automatically format your html/ts etc code so you dont have to. which basically means all devs code will look consistent and identical which is great when working in a team and therefore means the pr comments are more about the code and rather not the formatting.

you dont have to use prettier with git or command line etc. feel free to use in any local way that suits you best. might even be possible to add a npm script which you could run to invoke prettier and linting together?

Reply:
done. removed ignore.